src/components/Clothing.js

```diff
@@ -1,12 +1,15 @@
 import { useState } from 'react'
 import styled, { css } from 'styled-components/macro'
+import Bookmark from './Bookmark'
 
 function Clothing({ title, color, pattern, material, fitting, imageUrl }) {
   const [showDetails, setShowDetails] = useState(false)
 
   return (
     <StyledSection showDetails={showDetails}>
+      <Bookmark />
       <StyledButton
+        aria-label="clothing-button"
```

Comment:
This should be descriptive of what the button is doing: 
```suggestion
        aria-label={showDetails ? 'Hide details' : 'Show details'}
```